src/CommunityToolkit.Maui/Converters/ColorToComponentConverter.shared.cs

```diff
@@ -8,7 +8,12 @@ namespace CommunityToolkit.Maui.Converters;
 public class ColorToByteAlphaConverter : BaseConverterOneWay<Color, byte>
 {
 	/// <inheritdoc/>
-	public override byte ConvertFrom(Color value) => value.GetByteAlpha();
+	public override byte ConvertFrom(Color value)
```

Comment:
Can we mark Color as nullable?

Reply:
I think it makes more sense to keep `Color` as not Nullable. 

For example, it doesn't make sense to convert `null` to `byte`